b-real-time-htb.js

```diff
@@ -150,12 +157,28 @@ function BRealTimeHtb(configs) {
          */
 
         /* PUT CODE HERE */
+        
+        var __tags = [];
+        for(var i=0;i<returnParcels.length;i++) {
+          var returnParcel = returnParcels[i], tag = {}, uuid = System.generateUniqueId();
+          tag.sizes = returnParcel.xSlotRef.sizes.map(function(obj) {
+            return {width:obj[0], height:obj[1]};
+          });
+          tag.id = returnParcel.xSlotRef.placementId;
+          tag.primary_size = tag.sizes[0];
+          tag.allow_smaller_sizes = false;
+          tag.prebid = true;
+          tag.disable_psa = true;
+          tag.uuid = uuid;
+          returnParcel.uuid = uuid;
+          __tags.push(tag);
+        }
 
         /* -------------------------------------------------------------------------- */
-
+        
         return {
             url: baseUrl,
-            data: queryObj,
+            data: {tags: __tags},
             callbackId: callbackId
```

Comment:
Please add the following to the returned object: 
`networkParamOverrides: {method: 'POST'}`

```
return {
    url: baseUrl,
    data: {tags: __tags},
    callbackId: callbackId,
    networkParamOverrides: {method: 'POST'}
}
```